sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Constants.java

```diff
@@ -99,6 +99,7 @@ public static final class Properties {
 
         //Indexing Policy.
         public static final String INDEXING_POLICY = "indexingPolicy";
+        public static final String OVERWRITE_POLICY = "overwritePolicy";
```

Comment:
Please remove - should not be needed anymore

Reply:
Done.